packages/manager/modules/billing/src/main/history/billing-main-history.controller.js

```diff
@@ -10,13 +11,18 @@ export default class BillingMainHistoryCtrl extends ListLayoutHelper.ListLayoutC
     $uibModal,
     Alerter,
     atInternet,
+    coreConfig,
     ouiDatagridService,
   ) {
     super($q, ouiDatagridService);
     this.$translate = $translate;
     this.$uibModal = $uibModal;
     this.Alerter = Alerter;
     this.atInternet = atInternet;
+
+    this.displayHtmlInvoice = !HIDE_HTML_INVOICE_FOR_COUNTRIES.includes(
+      coreConfig.getUser()?.country,
```

Comment:
I think we should use `ovhSubsidiary` instead of `country`.
Are you sure we must display invoices accourding `country` ?

Because we can edit `country` field into account edit page.

Reply:
Yes, we need to display based on the `country`. Because for Indian customers, we cannot rely on `ovhSubsidiary`.

Reply:
Ok, thanks